### ISSUE TITLE
Proper fork

### ISSUE THREAD

**Issue by ligi** [s-matthew-english/goethe_0.1#1]
This should be properly forked from go-ethereum - removing the whole history is a bad thing

**Comment by s-matthew-english** [comment]
what do you mean? I'm trying to change it to look at the opcode execution time like this: 

https://ethereum.stackexchange.com/questions/23858/evaluation-of-execution-speeds-for-various-opcodes-in-evm

https://ethereum.stackexchange.com/questions/23917/log-output-of-modified-function-to-geth-console

**Comment by ligi** [comment]
This is OK - but still should be done on a fork. You removed the whole history of the project - see:
https://github.com/s-matthew-english/goethe_0.1/commits/master

This should not only be 2 commits but  > 8000

**Comment by s-matthew-english** [comment]
I just did that so I could easily set it up with my personal version on upstream, and then I was thinking I would update it from the normal goethe repo periodically, why does it matter? I'm not trying to be indolent, I've never thought about this as an issue before

**Comment by ligi** [comment]
I think you should do a proper fork - and now it's not yet too late

**Comment by s-matthew-english** [comment]
yeah sure, I guess it makes sense, I'll do it later this afternoon.

**Comment by s-matthew-english** [comment]
man, actually I don't think it makes sense to make it a fork, because I already have a fork of geth, which I keep updated with the network, but this one I want to change in ways that won't be able to be merged back in

**Comment by ligi** [comment]
then make a branch on the fork

**Comment by s-matthew-english** [comment]
is this the right way? 

https://github.com/s-matthew-english/go-ethereum/tree/goethe_0.0

**Comment by ligi** [comment]
yes!-)

**Comment by s-matthew-english** [comment]
cool, so- I'm going to delete this repo
